text_collectors/boots.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+#
+# Description: Expose metrics for number of boots of the system
+# Dependencies: none
+#
+# The script creates one metric series which is the number of recorded
+# boots in the /var/log/wtmp file. If wtmp is rotated, the next time the
+# node exporter is restarted it will recalculate the value, and may over
+# count restarts by one.
+#
+# Author: Clayton Coleman <smarterclayton@gmail.com>
+
+set -euo pipefail
+
+v="${TMPDIR:-/tmp}/boots.working"
```

Comment:
The collector is containerized (right?) but I still try hard to avoid predictable filenames in `/tmp` because if the code ever runs directly on a host system it can be a problem.  Any reason not to just do e.g.:
```
v=$(mktemp --suffix=.boots.working)
```

That line replaces `rm -f` and `touch` too.

Reply:
We're consistent with the other script, and we create tmp explicitly.  I'm not averse to doing that in a follow up (for general style things)

Reply:
Added it as a second commit